pulp_smash/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -101,72 +103,98 @@ def test_importer_config(self):
                 self.assertEqual(importers[0][key], body['importer_' + key])
 
 
-class SyncValidFeedTestCase(utils.BaseAPITestCase):
-    """Create an RPM repository with a valid feed and sync it.
+# This class is left public for documentation purposes.
+@unittest2.skip('Base test case.')
+class SyncRepoBaseTestCase(utils.BaseAPITestCase):
```

Comment:
We could also place this class in `pulp_smash.utils`. I didn't do that b/c the placement seems odd. This class's `setUpClass` and `test_task_progress_report` methods are both specific to the RPM plugin.